decidim/decidim
# Fix autocomplete result list duplicates

#### :tophat: What? Why?
When searching for participants in a new conversation, the autocomplete list gave duplicate results for one user group if the search term matched both name and nickname of the user group. To fix this we added a filter to the result list, so it only shows 1 result of the two with the same nickname.

#### :pushpin: Related Issues
*Link your PR to an issue*
This problem has already been reported in the github of the autocomplete library:
https://github.com/TarekRaafat/autoComplete.js/issues/102
https://github.com/TarekRaafat/autoComplete.js/issues/314

#### Testing
1. Create a user group as any user and give it a name and nickname that match at least two consecutive characters(in Decidim the default amount of letters needed for the autocomplete to give results is two)
2. Now after creating the user group go to "Conversations" page
3. Create a new conversation
4. Search for the user group with those 2 consecutive characters
5. You should see two results for your user group

### :camera: Screenshots
Before fix:
![image](https://github.com/decidim/decidim/assets/110532525/53f6e199-bf55-43b4-8354-3ff6cf322d97)

:hearts: Thank you!
